views/index.hbs

```diff
@@ -1,3 +1,7 @@
+<div class="container">
 <div>
-
+<img src="/images/beer.png" alt="beer">
+<a href="/">Check the Beers!</a>
+<a href="/">Check a Random Beer!</a>
```

Comment:
This links are both sending you to the home page, you need to fill the href path:
```
<a href="/beers">Check the Beers!</a>
<a href="/random-beer">Check a Random Beer!</a>
```